crates/languages/src/rust.rs

```diff
@@ -426,6 +426,20 @@ impl ContextProvider for RustContextProvider {
                 tags: vec!["rust-mod-test".to_owned()],
                 ..TaskTemplate::default()
             },
+            TaskTemplate {
+                label: format!(
+                    "cargo run -p {}",
+                    RUST_PACKAGE_TASK_VARIABLE.template_value()
+                ),
+                command: "cargo".into(),
+                args: vec![
+                    "run".into(),
+                    "-p".into(),
+                    RUST_PACKAGE_TASK_VARIABLE.template_value(),
```

Comment:
Nit: we should also pass a name of the current file without file extension, as there may be multiple bin target for a single package;
tl;dr: Instead of `cargo run -p RUST_PACKAGE_TASK_VARIABLE`, I'd do `cargo run -p RUST_PACKAGE_TASK_VARIABLE --bin ZED_STEM`

Reply:
Thank you. I indeed overlooked that point and have updated it accordingly.

```bash
cargo run -p RUST_PACKAGE_TASK_VARIABLE --bin ZED_CUSTOM_RUST_BIN_NAME
```